internal/provider/provider.go

```diff
@@ -77,9 +84,18 @@ func (p *EtleapProvider) Configure(ctx context.Context, req provider.ConfigureRe
 		Password: password,
 	}
 
+	transport := &http.Transport{
+		TLSClientConfig: &tls.Config{
+			InsecureSkipVerify: data.SkipCertificateValidation.ValueBool(),
```

Comment:
This will return `false` if undefined, so we don't have to do a null check here.